examples/morpheusvm/controller/controller.go

```diff
@@ -78,8 +84,7 @@ func (*factory) New(
 		return nil, nil, nil, nil, nil, nil, nil, nil, err
 	}
 
-	c.snowCtx.Log.SetLevel(c.config.LogLevel)
```

Comment:
I don't know why we do this - I think we should just use whatever log level avalanchego uses.

Reply:
Makes sense to me, we should remove `LogLevel` from the config in this case